src/schedule-and-details/introducing-section/introduction-video/IntroductionVideo.scss

```diff
@@ -0,0 +1,11 @@
+.embed-video-container {
+  @include pgn-box-shadow(1, "down");
+
+  box-sizing: border-box;
```

Comment:
It seems to me that we have a global declaration of this property for all child elements on the page? Please make sure that this property is really needed.

Reply:
Removed